resources/public/api/conf/1.0/schemas/BonusPaymentRequest.post.schema.json

```diff
@@ -69,8 +69,28 @@
           "claimReason"
         ]
 
+      },
+      "supersede" :
+      {
+        "description": "Supresede bonus payment by requesting for an additional bonus amount or recover an overpaid amount",
+        "type": "object",
+        "properties" :
+        {
+          "automaticRecoveryAmount":{"description": "This is used to confirm the amount due to HMRC from the LISA manager. If the superseded transaction result is a less than zero, you must give the automatic recovery amount. This is so that HMRC can calculate the amount to collect from the LISA manager. Leave it blank if there are no funds available. HMRC will recover overpaid funds from the LISA investor.", "type" : "number"},
```

Comment:
You can't leave this blank, it's a required field. If there are no funds available, the automatic recover amount is 0.

Reply:
I'd also be inclined to order this field after the "supersededTransactionResult" in the documentation, rather than right at the top. Reading through these fields for the first time, this is a very confusing field to try to understand without having the context of having read the fields below it first.